data/core/commandview.lua

```diff
@@ -275,17 +276,31 @@ end
 
 
 function CommandView:update_suggestions()
-  local t = self.state.suggest(self:get_text()) or {}
+  local text = self:get_text()
+  local t = self.state.suggest(self.last_change == "suggestion" and "" or text) or {}
```

Comment:
It might be better to save the user-inserted text, otherwise:
1. The user starts writing something
2. Suggestions pop up
3. User moves to the next suggestion
4. Suggestions "reset"

At the moment this seems to work because we're special-casing `""` inside `core:find-file`.

Reply:
How about that?

Reply:
Initialize it somewhere and we've got a deal.

Reply:
Oh yeah, initialize `self.last_change` in `:new` too.

Reply:
Done.